docker-compose/cspcbfmcs-docker-compose.yml

```diff
@@ -4,7 +4,7 @@ volumes:
 
 services:
   cbfmaster:
-    image: ${DOCKER_REGISTRY_HOST}/ska-docker/mid-cbf-mcs:0.1.0
+    image: ${DOCKER_REGISTRY_HOST}/ska-docker/mid-cbf-mcs:0.1.2
```

Comment:
No for immediate action, but we can consider sending the image version as a separate parameter to the docker-compose so that changing the version at a single place will result in minimizing the manual changes at multiple places.

Reply:
Sure!

Reply:
Commit ID: 9ab1a8bbeec93788ee716d8c14cabd8cb9e7a1bd